### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by ibeltagy** [rafaelbailo/betterposter-latex-template#1]
Thank you @rafaelbailo for converting Mike Morrison's design into latex. I used it to design my poster (presented a few days ago) and it worked well. It took like a day to make, and you can check it here: https://www.overleaf.com/read/fjdkxtgprkjg

Below are a few comments about my experience designing it: 

Design comments: 
- I found the title, name and affiliations to work better in the middle column because that's where people expect them, and also to save space on the left column.
- The template doesn't work if you have an important horizontal figure or table. I tried to scale down the figure (the upper left one), and it fits but it is not great. Also this won't work at all if the figure was wider or more important. 
- The phone and qrcode is larger than you want.

Latex comments: 
- Font sizes are on a scale that's not intuitive. I usually rely on \normal, \large, \huge ... to pick a font size, but here you need to use numeric sizes because even \HUGE is not large enough, and it is are not easy to decide what numeric size to use
- I made a lot of changes in the `.cls` file to change sizes, alignments, .. etc. If this is usually the case, then it is better to move more of the stuff in the `.cls` file into the `.tex` file.
- Please make it easy to set poster dimension for printing. Popular sizes are 36x24 and 48x36

Thanks a lot for this helpful template.

**Comment by ogourgue** [comment]
I fully agree that the management of font sizes is not optimal, and I would suggest a revision on that. For the rest, I found the template great and very useful! Thanks a lot!
